src/main.rs

```diff
@@ -115,6 +122,15 @@ fn main() {
             let url = get_url(&src, &host, &compiler, &args);
             println!("URL: {}", url);
         }
+        if matches.is_present("open") {
+            let url = get_url(&src, &host, &compiler, &args);
+            let res = open::that(url).is_ok();
```

Comment:
This is a perfect place to take advantage of Rust's `match` expressions.

I might suggest something like:
```
match open::that(url) {
    Ok(_) => println!("Opened result in browser"),
    Err(e) => println!("Failed to open the compilation in the browser."),
};
```

Reply:
That's awesome! Thank you for teaching me. I will push an update.